mypy/checker.py

```diff
@@ -1923,6 +1934,10 @@ def lvalue_type_from_base(self, expr_node: Var,
         if base_var:
             base_node = base_var.node
             base_type = base_var.type
+
+            if isinstance(base_node, TypeAlias):
+                return None, base_node
```

Comment:
Slightly unrelated: Could you update the docstring to describe the return value? Now it's not immediately clear why this is correct.